archiver/archive.go

```diff
@@ -53,6 +55,11 @@ func ArchivePackage(w io.Writer, properties PackageProperties) (err error) {
 
 	rootDir := fmt.Sprintf("%s-%s", properties.Name, properties.Version)
 
+	err = writePackageIndexToArchive(properties.Path, rootDir, tarWriter)
```

Comment:
Could you add a note here that we have this in because Kibana still relies on it but it should potentially be removed in the future?

Reply:
I left a note